packages/flutter_tools/test/general.shard/update_packages_test.dart

```diff
@@ -1,91 +0,0 @@
-// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Is there anyway we can test the new command somewhere?  Convert it into a smoke test?  `update-packages` isn't just a cute CI or development tool, if it's doing something wonky it can break The World.

Reply:
Its tested on every CI bot (its how they download the deps), on benchmarks (technical debt, dependency measurement), and multiple times a week on an upgrade.

Reply:
Okay good point.